examples/ancestral-structure/index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang='en'>
+
+<head>
+  <meta charset="utf-8">
+
+  <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+  <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap-theme.min.css">
+  <link href="//veg.github.io/phylotree.js/phylotree.css" rel="stylesheet">
+
+  <script src="//code.jquery.com/jquery.js"></script>
+  <script src="//netdna.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+  <script src="//d3js.org/d3.v3.min.js"></script>
+  <script src="/phylotree.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js" charset="utf-8"></script>
+  <script type='text/javascript' src='bio-pv.min.js'></script>
```

Comment:
We should get away from using CDNs and perhaps use webpack.

Reply:
I think most of our users will be more familiar with classic Javascript development practices like this. So if we really want to get away from CDNs, I'd include the source in our repository before I converted each example to webpack. I do think a webpack example would be useful

Reply:
Good point. I forgot that we want users to be able to get started with examples quickly for a moment.